### DISCUSSION TITLE
Chai as a dependency?

### DISCUSSION THREAD

**Discussion by NickWoodward** [chaijs/chai#1497] [General]
Just wondering why Chai is a dependency rather than a dev dependency? Might well be a misunderstanding on my part as I tend to wonder this a lot (mocha/sinon/various other packages) :D

**Comment by keithamus** [comment]
You should install it as a development dependency. Is there documentation which suggests otherwise?

**Reply by NickWoodward** [reply]
Hi Keith,

Ah, I guess there isn't when you actually read past the first installation line. My bad! Still, this threw me for a second, but it's certainly not limited to Chai.

npmjs:

![image](https://user-images.githubusercontent.com/16296682/213541437-0f32860a-2caa-4957-981c-f28c9cdcef61.png)

official docs:

![image](https://user-images.githubusercontent.com/16296682/213541520-34c24e3d-331b-4a83-ad0b-047d3fd86e3c.png)

**Reply by NickWoodward** [reply]
Fully admit that's selective screenshotting! Thanks for clarifying though
